env0/resource_template_test.go

```diff
@@ -502,7 +502,7 @@ func TestUnitTemplateResource(t *testing.T) {
 						"repository": template.Repository,
 						"type":       "gruntyform",
 					}),
-					ExpectError: regexp.MustCompile(`must be one of: terragrunt, terraform`),
+					ExpectError: regexp.MustCompile(`must be one of`),
```

Comment:
Does this work? shouldn't it list all of the available types

Reply:
It works. I didn't want us to modify the test every time a new type was added.
(This test isn't beneficial, to begin with).

Since it's a regex match, it passes because the prefix never changes.